client/extensions/woocommerce/app/dashboard/manage-orders.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import { getLink } from 'woocommerce/lib/nav-utils';
+import ProcessOrdersWidget from 'woocommerce/components/process-orders-widget';
+import ReadingWidget from 'woocommerce/components/reading-widget';
+import ShareWidget from 'woocommerce/components/share-widget';
+
+class ManageOrders extends Component {
+	static propTypes = {
+		site: PropTypes.shape( {
+			slug: PropTypes.string.isRequired,
+		} ),
+	};
+
+	renderProcessOrdersWidget = () => {
+		return (
+			<ProcessOrdersWidget className="dashboard__process-orders-widget" />
+		);
+	}
+
+	possiblyRenderShareWidget = () => {
+		// TODO - connect to display preferences in a follow-on PR
+		const { site, translate } = this.props;
+		return (
+			<ShareWidget
+				text={ translate( 'Share a link to your store on social media.' ) }
+				title={ translate( 'Share your store' ) }
+				urlToShare={ site.URL }
+			/>
+		);
+	}
+
+	possiblyRenderReadingWidget = () => {
+		// TODO - connect to display preferences in a follow-on PR
+		const { translate } = this.props;
+		return (
+			<ReadingWidget
+				text={ translate( 'You’re not alone! Get tips from seasoned merchants,' +
+					' learn best practices to keep your store ship-shape,' +
+					' and find how to boost your sales and drive traffic.' ) }
+				title={ translate( 'Recommended reading' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **21** times:
`translate( 'Recommended Reading' )` *ES Score: 22*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/store-dashboard-fledgling)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).